src/app/containers/Timestamp/relativeTimestamp.js

```diff
@@ -0,0 +1,31 @@
+const relativeTime = receivedTimestamp => {
+  const currentTime = Date.now();
+  const timeDifferenceMilliseconds = currentTime - receivedTimestamp;
+  const toMinuteDivisor = 1000 * 60;
+  const timeDifferenceMinutes = parseInt(
+    timeDifferenceMilliseconds / toMinuteDivisor,
```

Comment:
```suggestion
    timeDifferenceMilliseconds / (1000 * 60),
```

I'd avoid the one-off `const` here - its name is quite tricky to mentally parse 🤔

Reply:
sure, went back and forth with these because I was trying to get codeclimate to reduce the cognitive complexity, but it turns out that's just a result of having multiple operands 😭